data/snippets.conf

```diff
@@ -58,6 +66,11 @@ try=try%block%\ncatch (%cursor%)%block_cursor%
 if=if (%cursor%)%block_cursor%
 else=else%block_cursor%
 for=for (int i = 0; i < %cursor%; i++)%brace_open%\n%brace_close%
+syso=System.out.println("%cursor%");
+Syso=System.out.println("%cursor%");
+main=package %cursor%;\n\npublic class Main {\n\tpublic static void main(String[] args) {\n\t\t%cursor%\n%brace_close1%%brace_close%
```

Comment:
This `main` snippet doesn't use the same style as the other one, like `do` just below: this one has the opening brace on the class/method line, and the ones below have it on the next one.  This should be consistent.